data/fro/Zink_1984_Cleriadus/stemma.gv

```diff
@@ -0,0 +1,34 @@
+digraph {
+
+O -> x; 
+O -> y;
+
+y -> P3;
```

Comment:
I find the way this author has drawn his stemma can be ambiguous. Does P3, L and T all descend straight from y, or do P3 and L have a common direct ancestor ?

Reply:
The common ancestor of P3L is the _deperditus_ y. T shares several errors with the manuscripts of both the groups (x and y), but it also has a large amount of individual errors. It is therefore contaminated.